deploy/olm/csv-rhel/storageos.clusterserviceversion.yaml

```diff
@@ -394,7 +394,21 @@ spec:
           added to the StorageOS role.
         displayName: Disable Fencing
         path: disableFencing
-      - description: When enabled, the Operaaor will not perform any actions on the
+      - description: Disable TCMU can be set to true to disable the TCMU storage
```

Comment:
I think the descriptions have gotten mixed up here. This should describe `disableFencing`.

Reply:
The order is correct, but not clear since the description is the 1st field.